examples/tv-casting-app/darwin/args.gni

```diff
@@ -20,7 +20,7 @@ chip_device_project_config_include = "<CHIPProjectAppConfig.h>"
 chip_project_config_include = "<CHIPProjectAppConfig.h>"
 chip_system_project_config_include = "<SystemProjectConfig.h>"
 
-chip_project_config_include_dirs += [
+chip_project_config_include_dirs = [
```

Comment:
Should this file not be importing `${chip_root}/config/standalone/args.gni` instead of `config/ios/args.gni`, and this line can remain `+=` ?

Reply:
I have updated this file to use `config/darwin/args.gni`.
And the line still need to be changed from `+=` to `=` since there is no definition of `chip_project_config_include_dirs` there.